lib/archethic/metrics/metric_client.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ArchEthic.Metrics.MetricClient do
+  @moduledoc """
+  gensever
```

Comment:
If you can put a more meaning full description, it will be better, I think

Reply:
I did but I removed due to urgency of newsletter